src/core.jsx

```diff
@@ -61,14 +72,19 @@ export default (
     }
 
     componentWillMount() {
-      // Load from props
-      if (this.omitLoadInProps(this.props)) {
-        this.props.load()
-      }
+      let loadFunctionName = 'load'
 
       // Load from hoc argument
```

Comment:
delete this code block, and handle it in one block code with `loadFunctioName || 'load'`, you can add a function to factorise the code